pyproject.toml

```
@@ -7,7 +7,7 @@ authors = [
 ]
 description = "Python toolkit for analysis, visualization, and comparison of spike sorting output"
 readme = "README.md"
-requires-python = ">=3.9,<4.0"
+requires-python = ">=3.9,<3.13"  # Only numpy 2.0 supported on python 3.13 for windows. We need to wait for fix on neo
```

```suggestion
requires-python = ">=3.9,<3.13"  # Only numpy 2.1 supported on python 3.13 for windows. We need to wait for fix on neo
```

Actually if we want to be super accurate (when I explained this I did a lazy job), it is actually 2.0 is good for python 3.9-3.12 and then 2.1 is when 3.13 support was added. Really based on my neo testing there was no issue adding 2.1 at the exact same time as 2.0, so this doesn't practically matter, but if we want the comment to be accurate then technically we should say 2.1. Although based on the Neo CI 3.13 did work with 2.0 on Linux it just isn't officially supported.

---

Thanks. I read the issue on numpy that you linked on neo. It left me a bit confused about the whole thing. Might have to check it twice.